nshm_geojson_fault_parser.py

```diff
@@ -13,31 +13,126 @@
 app = typer.Typer()
 
 
-def strike_between_coordinates(a: (float, float), b: (float, float)) -> float:
-    a_lat, a_lon = a
-    b_lat, b_lon = b
+def strike_between_coordinates(
```

Comment:
I know this PR doesn't add this function, but I think this function isn't actually needed? It just extracts values from a tuple and calls another function? And its only used once as far as I can tell?
I think this can also be simplified to
```python
qcore.geo.ll_bearing(*point_a_coords[::-1], *point_b_coords[::-1])
```